apps/muebel_splunk_all_indexes/local/indexes.conf

```
@@ -75,3 +75,6 @@ datatype = metric
 [storage_data]
 frozenTimePeriodInSecs = 1000000
 
+[newtork_data]
```

```suggestion
[network_data]
```
This looks like a typo